http/headers/feature-policy.json

```diff
@@ -1853,6 +1853,56 @@
             }
           }
         },
+        "web-share": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Feature-Policy/web-share",
+            "support": {
+              "chrome": {
+                "version_added": null
+              },
+              "chrome_android": {
+                "version_added": null
+              },
+              "edge": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": "81",
+                "notes": "Firefox 81 enables support for the Feature-Policy web-share, but does not support the WebShare API. See <a href='https://bugzil.la/1653481'>bug 1653481</a>."
+              },
+              "firefox_android": {
+                "version_added": "81",
+                "notes": "Firefox 81 enables support for the Feature-Policy web-share, but does not support the WebShare API. See <a href='https://bugzil.la/1653481'>bug 1653481</a>."
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": null
+              },
+              "opera_android": {
+                "version_added": null
+              },
+              "safari": {
+                "version_added": null
+              },
+              "safari_ios": {
+                "version_added": null
+              },
+              "samsunginternet_android": {
+                "version_added": null
+              },
+              "webview_android": {
+                "version_added": null
+              }
+            },
+            "status": {
+              "experimental": true,
```

Comment:
FYI this spec is draft still.  This is no longer marked as experimental in Mozilla code though.

Reply:
#6905 is open to cover this better, but "experimental" in BCD is likely to mean "only has one implementation" (or near enough to it) in the future. This is value is good for now! 👍